packages/sdk/src/conversation/interface.ts

```diff
@@ -313,3 +412,21 @@ export interface ConversationOptions {
     enabled?: boolean;
   };
 }
+
+/**
+ * Interface for user to customize sso execution activity handler
+ */
+export interface SsoExecutionActivityHandler extends TeamsActivityHandler {
+  /**
+   * Add {@link TeamsFxBotSsoCommandHandler} instance to {@link SsoExecutionDialog}
+   * @param handler {@link SsoExecutionDialogHandler} callback function
```

Comment:
we also implemented many other functions like `handleTeamsSigninVerifyState`. should we add all these functions to this interface?

Reply:
sure